wiki/People/Performance_Points_Committee/en.md

```diff
@@ -7,36 +7,81 @@ tags:
+- Organizing development efforts and helping contributors
+- Reviewing and developing changes in difficulty calculation algorithms
+- Deciding which changes will be deployed and used in the game
+- Assisting with writing news posts for newly deployed changes
 
-The osu!taiko panel was formed on 1 May 2022 in response to the same survey as above, with [major changes to the star rating algorithm](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit).
+## Workflow
+
+Whenever a new difficulty calculation proposal is created on [GitHub](https://github.com/ppy/osu), the Performance Points Committee relevant to the ruleset it's for will be tagged with a review request. After that members of the committee must review and test the proposal and write a review on its GitHub page. When there are enough committee approvals the change is considered ready for deployment and is passed to osu! developers for final code inspection before merging into the game.
+
+When there are enough merged proposals the osu! developers will start preparing for a performance points deployment which will recalculate all existing scores using the new algorithms. Performance Points Committees are tasked with assisting with writing a news post about the upcoming changes.
+
+## Communication
+
+Performance Points Committees mainly operate on the [Performance Points Discord server](https://discord.gg/aqPCnXu) with some committees having separate servers for their respective ruleset. Committees also maintain a [Twitter account](https://x.com/pp_committee) that is being used for official announcements and difficulty calculation related news. You can also contact any of the members of the committees privately through osu! private messages.
 
 ## Members
 
+Members of the committees are hand-picked by osu! developers and/or other committee members and consist of people with a lot of experience working on the difficulty calculation system.
+
 ### osu!
 
-| osu! profile | GitHub profile |
-| :-- | :-- |
-| ::{ flag=GB }:: [moonpoint](https://osu.ppy.sh/users/9558549) | [apollo-dw](https://github.com/apollo-dw) |
-| ::{ flag=RU }:: [StanR](https://osu.ppy.sh/users/7217455) | [stanriders](https://github.com/stanriders) |
-| ::{ flag=GB }:: [tsunyoku](https://osu.ppy.sh/users/11315329) | [tsunyoku](https://github.com/tsunyoku) |
+The osu! panel was first announced to the public through the [November 2021 *Performance Points & Star Rating Updates* news post](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to a [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) held earlier that year.
+
+| osu! profile | GitHub profile | Join date |
+| :-- | :-- | :-- |
+| ::{ flag=GB }:: [moonpoint](https://osu.ppy.sh/users/9558549) | [apollo-dw](https://github.com/apollo-dw) | November 9, 2021 |
```

Comment:
use date format same as catch and mania

alternatively, YYYY-MM-DD in tables for easier columnwise comparison